flixel/input/gamepad/FlxGamepadManager.hx

```diff
@@ -368,7 +368,7 @@ class FlxGamepadManager implements IFlxInputManager
 	@:allow(flixel.FlxG)
 	private function new() 
 	{
-		#if FLX_OPENFL_JOYSTICK_API
+		#if (FLX_OPENFL_JOYSTICK_API && !next)
```

Comment:
Just add that condition to `FlxConditionals` where `FLX_OPENFL_JOYSTICK_API` is defined.